app/services/witter.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function weet() {
+  try {
+    const response = await axios.get('https://quote-garden.herokuapp.com/api/v3/quotes/random');
```

Comment:
The URL of the external API should be saved in an environment variable in the `.env` file and then add it in `config/index.js` in the `config` variable.